bin/dump-to-git.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var fs = require('fs');
+var process = require('process');
+var child_process = require('child_process');
+var Promise = require('bluebird');
+var NodeGit = require('nodegit');
+var LZString = require('lz-string');
+var models = require("./lib/models");
+
+var repoRootDir = './notes';
+
+process.chdir(repoRootDir);
+
+
+function exportNotes() {
+  return models.Note
+  .findAll()
+  .then(function (notes) {
+    // Iterate over all notes
+    return Promise.all(notes.map(function (note) {
+      var file = LZString.compressToBase64(note.id);
```

Comment:
one could also try to parse the optional yaml metadata at the top of a note. Or use the first headline.

Only downside is, that they may not be unique. But you'd get readable filenames in return.